packages/devtools_app/lib/src/screens/network/network_controller.dart

```diff
@@ -356,10 +359,8 @@ class CurrentNetworkRequests {
   ///
   void updateOrAdd(
     HttpProfileRequest request,
```

Comment:
super nit: this trailing comma can be removed now that this method signature will fit on a single line

Reply:
Done.